service/src/main/java/bio/terra/workspace/app/configuration/external/BufferServiceConfiguration.java

```diff
@@ -58,11 +70,26 @@ public String getClientCredentialFilePath() {
   }
 
   public String getAccessToken() throws IOException {
-    try (FileInputStream fileInputStream = new FileInputStream(clientCredentialFilePath)) {
-      GoogleCredentials credentials =
-          ServiceAccountCredentials.fromStream(fileInputStream).createScoped(BUFFER_SCOPES);
-      AccessToken token = credentials.refreshAccessToken();
-      return token.getTokenValue();
+    try {
+      if (features.isAzureControlPlaneEnabled()) {
+        TokenCredential credential = new DefaultAzureCredentialBuilder().build();
```

Comment:
Is the RBS being spun up in the azure control plane? If not, I'm wondering if we should think about throwing an exception if an attempt is made to use the RBS in azure.